system-addon/content-src/components/TopSites/_TopSites.scss

```diff
@@ -339,6 +331,58 @@ $half-base-gutter: $base-gutter / 2;
       text-align: right;
     }
 
+    .url {
+      input:dir(ltr) {
+        padding-right: 32px;
+      }
+
+      input:dir(rtl) {
+        padding-left: 32px;
+      }
+    }
+
+    .enable-custom-image-input {
+      display: inline-block;
+      font-size: 13px;
+      margin-top: 4px;
+      text-decoration: underline;
```

Comment:
Does this actually want underline? Our other link-like things don't seem to have that styling.

Reply:
That's how the spec is.. or was?
<img width="643" alt="image" src="https://user-images.githubusercontent.com/810040/36721976-b03719a2-1bac-11e8-9e8e-805013773e8a.png">

Reply:
I was also trying to get a recent spec last week, and aaron said he sent them to you on slack?

Reply:
I can double check with Aaron.

Reply:
Bryan said this should always be blue whether hovered or not. And no underline normally but underlined on hover.

Reply:
Removed the underline.